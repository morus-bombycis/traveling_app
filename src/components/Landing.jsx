import React from 'react';
import { Link } from 'react-router-dom';
import CarouselCities from './CarouselCities'

const Landing = () => {
    return (
        <div>
            <div>
                <div>
                    <h1>MYtinerary</h1>
                    <img src="/MytineraryLogo.png" height="50" />
                    <p>Find your perfect trip, designed by insiders who know and love their cities.</p>
                </div>
                <div>
                    <Link to="/cities">
                        <img src="/circled-right-2.png" width="50" />
                    </Link>
                </div>
                <div>
                    <CarouselCities />
                </div>
            </div>
        </div>
    );
}

export default Landing;
