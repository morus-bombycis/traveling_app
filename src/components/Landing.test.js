import React from 'react';
import { shallow } from 'enzyme';
import Landing from './Landing';

it('Landing component should render without crashing', () => {
    shallow(<Landing />);
});