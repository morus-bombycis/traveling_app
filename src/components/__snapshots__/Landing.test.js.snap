// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`landing component snapshot with Enzyme 1`] = `
<div>
  <h1>
    Landing Page
  </h1>
  <Link
    style={
      Object {
        "color": "#fff",
        "textDecoration": "none",
      }
    }
    to="/"
  >
    Home
  </Link>
   | 
  <Link
    style={
      Object {
        "color": "#fff",
        "textDecoration": "none",
      }
    }
    to="/"
  >
    About
  </Link>
</div>
`;
